### PR TITLE
Added Start commands

### PR DESCRIPTION
**Pull request by CodeWriterWriter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
+a![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
```
Comment (mcdonnelldean) [inline README.md]: little a needs to be removed

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
 # nodezoo-NPM-UPDATE
```
Comment (mcdonnelldean) [inline README.md]: all small caps

File: README.md [hunk 3]
```diff
@@ -14,6 +14,14 @@ Nodezoo.com micro-service handling npm data.. Please see the [main repo][] for m
 $ node npm-service.js --seneca.options.from=dev.options.js --seneca.log.all
 ```
 
+## Messages
+
+This micro-service recognizes the following messages:
```
Comment (mcdonnelldean) [inline README.md]: recognizes -> listens for

File: README.md [hunk 4]
```diff
@@ -1,21 +1,39 @@
-![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
+![nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
 
-# nodezoo-NPM-UPDATE
+## nodezoo-NPM-UPDATE
```
Comment (mcdonnelldean) [inline README.md]: No caps in name

File: README.md [hunk 5]
```diff
@@ -1,21 +1,39 @@
...
 Nodezoo.com micro-service handling npm data.. Please see the [main repo][] for more details.
 
 - __Sponsor:__ [nearForm][]
 
+## Install
+1. clone this repo into a root _/nodezoo_ folder.
+2. run `npm install`
 
-
```
Comment (mcdonnelldean) [inline README.md]: Missing tags section, doesn't match how nodezoo-stats does it

**Comment by mcdonnelldean** [comment]
You need to remove that mod/ stuff that got checked in. If it's all in the one folder just add mod to the .gitignore

File: package.json [hunk 6]
```diff
@@ -4,7 +4,14 @@
   "description": "nodezoo.com microservice handling npm updates.",
   "main": "npm-update.js",
   "author": "Richard Rodger (http://richardrodger.com)",
+  "contributors": [
+    "Isaac Mahon (github.com/CodeWriterWriter)"
+  ],
   "license": "MIT",
+  "scripts":{
+    "start": "node srv/update-dev.js ",
+    "start-dev": "node srv/update-dev.js "
```
Comment (mcdonnelldean) [inline package.json]: change to start:dev

File: README.md [hunk 7]
```diff
@@ -1,21 +1,50 @@
...
 ```
 
 ## Contributing
-The [NodeZoo][] org encourages open participation. If you feel you can help in any way, be it with documentation, examples, extra testing, or new features please get in touch.
+The [NodeZoo][] org encourages __open__ and __safe__ participation. If you feel you can help in any way, be it with documentation, examples, extra testing, or new features please get in touch.
+
+- Before contributing please review our __[Code of Conduct][CoC]__
 
 ## License
 Copyright (c) 2015, Richard Rodgers and other contributors.
```
Comment (mcdonnelldean) [inline README.md]: Rogers -> Roger

File: README.md [hunk 8]
```diff
@@ -1,21 +1,50 @@
-![Nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
+![nodezoo](https://raw.githubusercontent.com/rjrodger/nodezoo-web/to-redux/client/assets/img/logo-nodezoo.png)
 
-# nodezoo-NPM-UPDATE
+## nodezoo-npm-update
 Nodezoo.com micro-service handling npm data.. Please see the [main repo][] for more details.
```
Comment (mcdonnelldean) [inline README.md]: single full stop after npm data

File: README.md [hunk 9]
```diff
@@ -1,21 +1,50 @@
...
+## Install
+1. clone this repo into a root _/nodezoo_ folder.
+2. run `npm install`
 
+## Starting
+To start simply run,
+```
+npm start
+```
+### Tagging and Logs
```
Comment (mcdonnelldean) [inline README.md]: Linebreak

File: README.md [hunk 10]
```diff
@@ -1,21 +1,50 @@
...
 
+## Install
+1. clone this repo into a root _/nodezoo_ folder.
+2. run `npm install`
 
+## Starting
+To start simply run,
+```
+npm start
+```
```
Comment (mcdonnelldean) [inline README.md:19]: Linebreak

File: README.md [hunk 11]
```diff
@@ -1,21 +1,50 @@
...
+### Tagging and Logs
+To tag your service and set up logs simply pass the relevant switches on start,
+
+```
+npm start -- --seneca.options.tag=nodezoo-github --seneca.log.all
```
Comment (mcdonnelldean) [inline README.md]: Wrong tag for this lib

File: README.md [hunk 12]
```diff
@@ -1,28 +1,59 @@
...
+## Outbound Messages
+
+This micro-service emits no outbound messages.
+
+## Running with Curl
+
+Any of the messages above can be run using curl in the following format in the commanding
+```
+curl -d '{"role":"npm","task":"registry_subscribe"}' http://localhost:44005/act
 ```
```
Comment (mcdonnelldean) [inline README.md:43]: Add > **Note:** Ports are assigned automatically, please check the logs for the correct port to use.

File: README.md [hunk 13]
```diff
@@ -1,28 +1,61 @@
...
+
+This micro-service emits no outbound messages.
+
+## Running with Curl
+
+Any of the messages above can be run using curl in the following format in the command line
+```
+curl -d '{"role":"npm","task":"registry_subscribe"}' http://localhost:44005/act
 ```
+Note: Ports are assigned automatically, please check the logs for the correct port to use.
```
Comment (mcdonnelldean) [inline README.md]: Bold Note: Like this **Note:**

**Comment by mcdonnelldean** [comment]
@CodeWriterWriter can we get this updated? We need to remove CoC in favour of link, other than that it looks good.

**Comment by AdrianRossouw** [comment]
thanks @CodeWriterWriter

**Pull request merged**